apps/dashboard-api/src/controllers/events.controller.js

```
@@ -46,7 +43,9 @@ module.exports.track = async (req, res) => {
 
     return res.json({ success: true, data: {}, message: 'Event queued' });
   } catch (err) {
-    console.error('[events.controller] track error:', err);
-    return res.status(500).json({ success: false, data: {}, message: 'Internal server error' });
+    if (!(err instanceof AppError)) {
+      err = new AppError(500, 'Internal server error');
```

Biome (2.4.16)

[error] Reassigning a catch parameter is confusing.

(lint/suspicious/noCatchAssign)